packages/components/button/button.js

```diff
@@ -1,10 +1,21 @@
+const KEY_SPACE = 32
+const DEBOUNCE_TIMEOUT_IN_SECONDS = 1
+
 class Button {
   constructor($module) {
-    this.KEY_SPACE = 32
-    this.DEBOUNCE_TIMEOUT_IN_SECONDS = 1
+    if (!$module) {
```

Comment:
I find it a bit confusing that the element is called $module here... is it worth renaming it? I checked govuk-frontend and those components call this argument `$root` instead.